book/10-git-internals/sections/objects.asc

```diff
@@ -139,6 +139,13 @@ $ git cat-file -p master^{tree}
 The `master^{tree}` syntax specifies the tree object that is pointed to by the last commit on your `master` branch.
 Notice that the `lib` subdirectory isn't a blob but a pointer to another tree:
 
+[NOTE]
+====
+* Depending on what shell you use, you may encounter errors when using the `master^{tree}` syntax
+* In PowerShell or CMD on Windows, the `^` character is used for escaping, so you have to double it to avoid this: `git cat-file -p mater^^{tree}`
+* If you're using ZSH, the `^` character is used for globbing, so you have to enclose the whole expression in quotes: `git cat-file -p "master^{tree}"`
```

Comment:
Interesting. Why did you set this using bullets?

![image](https://user-images.githubusercontent.com/39902/37106023-8a1730d8-21e5-11e8-8a39-9c1074a39087.png)

Let's just write it as a single paragraph, I think that flows better.

Reply:
sure